### ISSUE TITLE
enemy bullets sometimes fade out when the enemy dies and then reappear

### ISSUE THREAD

**Issue by nasearle** [nasearle/fall-back#57]
possibly because of the `if (this.parentType === 'enemy')` line in BulletSprite on the client. when the enemy dies and is removed, this line is no longer true

**Comment by DavidScales** [comment]
yes but actually looks pretty cool. we could consider it a feature ;)
